### PR TITLE
Assume available positions functionality and remove feature flag

### PR DESCRIPTION
**Pull request by mjoyce91**
- Removes the `available_positions` flag and updates application logic to assume that `available_positions` is true
- Update tests to reflect this change

### CODE REVIEW

File: src/Components/SearchResultsExportLink/SearchResultsExportLink.jsx [hunk 1]
```diff
@@ -76,7 +73,7 @@ class SearchResultsExportLink extends Component {
   onClick() {
     const { isLoading } = this.state;
     const { isProjectedVacancy } = this.context;
-    const useExportEndpoint = getUseAP() || isProjectedVacancy;
+    const useExportEndpoint = isProjectedVacancy;
```
Comment (rtirserio) [inline src/Components/SearchResultsExportLink/SearchResultsExportLink.jsx]: This should always be true since `getUseAP()` would be true

Comment (mjoyce91) [reply src/Components/SearchResultsExportLink/SearchResultsExportLink.jsx]: Oh good point, nice catch

File: src/actions/filters/filters.js [hunk 2]
```diff
@@ -292,7 +290,7 @@ export function filtersFetchData(items = { filters: [] }, queryParams = {}, save
             // and we override what ever was done in the union prior to this block.
             // Here we map the AP cone/code model to the old model so that it plays nice
             // with our existing components.
-            if (item.item.description === 'skillCone' && getUseAP()) {
+            if (item.item.description === 'skillCone') {
```
Comment (mjoyce91) [inline src/actions/filters/filters.js]: Need to check these filters:
- Skills not displaying
- Post not displaying
- Functional bureaus showing undefined (undefined)

Comment (mjoyce91) [reply src/actions/filters/filters.js]: Fixed

**Pull request merged**